nomos/config.py

```diff
@@ -46,15 +46,20 @@ def get_tool_wrapper(self) -> ToolWrapper:
         :return: ToolWrapper instance.
         """
         tool_type, tool_name = self.tag.split("/", 1)
-        name = self.name or convert_camelcase_to_snakecase(
-            self.tool_name.split(".")[-1]
-        )
         tool_type = tool_type.replace("@", "")
+        if tool_type in ["mcp"]:
+            name = self.name or tool_name
```

Comment:
lets always force the user to use a name here. Add an assertion only is tool type is mcp and self.name is None